docs/assets/css/nafoarms.scss

```diff
@@ -124,6 +124,12 @@ body {
     padding-bottom: 2vw;
 }
 
+.redacted-tag {
+    background-color: balck;
```

Comment:
Black has a typo

Reply:
Oh actually this is from an unfinished thing, forgot to remove it

Reply:
should be fixed now